src/res_comments_table.ml

```diff
@@ -1310,9 +1311,36 @@ and walkExpression expr t comments =
         walkExpression callExpr t inside;
         after)
     in
-    let afterExpr, rest = partitionAdjacentTrailing callExpr.pexp_loc after in
-    attach t.trailing callExpr.pexp_loc afterExpr;
-    walkList (arguments |> List.map (fun (_, e) -> ExprArgument e)) t rest
+    if ParsetreeViewer.isJsxExpression expr then (
+      let props =
+        arguments
+        |> List.filter (fun (label, _) ->
+               match label with
+               | Asttypes.Labelled "children" -> false
+               | Asttypes.Nolabel -> false
+               | _ -> true)
+      in
+      let maybeChildren =
+        arguments
+        |> List.find_opt (fun (label, _) ->
+               label = Asttypes.Labelled "children")
+      in
+      match maybeChildren with
+      | None -> ()
```

Comment:
Is this not walking the props at all?
What happens if you have props with comments and no children?

Reply:
Would you give me a hint under what circumstances this would happen? I tested some cases and found children were always there

If there are no children elements inside, there will be `Pexp_construct "[]" ` in the AST tree
 ```
Labelled "children"
          expression (test.res[2,1+2]..[2,1+3])
            Pexp_construct "[]" (test.res[2,1+2]..[2,1+3]) ghost
            None
```

Reply:
How about adding a comment saying just that.